asf_tools/composite.py

```diff
@@ -81,23 +82,23 @@ def get_full_extent(raster_info: dict):
     return (ulx, uly), (lrx, lry), trans, proj
 
 
-def reproject_to_target(raster_info: dict, target_epsg_code: int, target_resolution: float) -> dict:
+def reproject_to_target(raster_info: dict, target_epsg_code: int, target_resolution: float, dir: str) -> dict:
     logging.info("Checking projections")
     target_raster_info = {}
     for raster, info in raster_info.items():
         epsg_code = get_epsg_code(info)
         resolution = info['geoTransform'][1]
         if epsg_code != target_epsg_code or resolution != target_resolution:
             logging.info(f"Reprojecting {raster}")
-            reprojected_raster = raster.replace('.tif', '.reproj.tif')
+            reprojected_raster = os.path.join(dir, os.path.basename(raster))
```

Comment:
This approach works, but might be too naive?
For reprojected files the log output looks like:
```
Processing raster /tmp/tmp2_zusrah/S1A_IW_20181110T162922_DVP_RTC30_G_gpuned_2345_VV.tif
Raster upper left: [102720.0, 6911430.0]; lower right: [409710.0, 6672900.0]
Reading raster values /tmp/tmp2_zusrah/S1A_IW_20181110T162922_DVP_RTC30_G_gpuned_2345_VV.tif
Reading area raster /tmp/tmp2_zusrah/S1A_IW_20181110T162922_DVP_RTC30_G_gpuned_2345_area.tif
Placing values in output grid at 12813:20764 and 0:10233
```
And for original files it looks like this (when I used `-i` for files all in the same directory):
```
Processing raster S1A_IW_20181112T161236_DVP_RTC30_G_gpuned_EE18_VV.tif
Raster upper left: [356820.0, 7019280.0]; lower right: [646380.0, 6802800.0]
Reading raster values S1A_IW_20181112T161236_DVP_RTC30_G_gpuned_EE18_VV.tif
Reading area raster S1A_IW_20181112T161236_DVP_RTC30_G_gpuned_EE18_area.tif
Placing values in output grid at 9218:16434 and 8470:18122
```